### PR TITLE
chore: update syft to v0.60.3

### PR DESCRIPTION
**Pull request by spiffcs**
## Summary
Consume https://github.com/anchore/syft/pull/1312 as the latest syft.

Current syft v0.60.2 has a small bug regarding PURL generation that makes some matching inconsistent for `TestCompareSBOMInputToLibResults`

The test images found for that integration will produce different results if grype is updated to consume `v0.60.2`.

🚧 Merge only when v0.60.3 has been released 🚧 

Note: the logger has not been fully refactored. Only the syft wrapper has been updated for compatibility. A follow-up PR will fully port grype over to the newest logger separately.

Signed-off-by: Christopher Phillips <christopher.phillips@anchore.com>

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -73,9 +76,16 @@ func initLogging() {
 	grype.SetLogger(logWrapper)
 
 	// add a structured field to all loggers of dependencies
-	syft.SetLogger(&logger.LogrusNestedLogger{
-		Logger: logWrapper.Logger.WithField("from-lib", "syft"),
-	})
+	syftLoggerCfg := logrus.Config{
+		EnableConsole: enableConsole,
+		FileLocation:  appConfig.Log.FileLocation,
```
Comment (kzantow) [inline cmd/cmd.go:81]: Is there a possibility having 2 loggers with the same file will end up overwriting or failing to open the file?

File: cmd/cmd.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/spf13/viper"
 	"github.com/wagoodman/go-partybus"
 
+	iface "github.com/anchore/go-logger"
```
Comment (kzantow) [inline cmd/cmd.go:14]: nit: maybe something with `log` or `logger` in the name? like `loggerAPI`? to make it a tiny bit more clear below

**Review by kzantow — APPROVED** [review]
Only question about having 2 different loggers with the same file reference

File: cmd/cmd.go (hunk 1 above)
Comment (spiffcs) [reply cmd/cmd.go]: Removed until we can get a PR that get's both syft and grype onto the same logger

File: cmd/cmd.go (hunk 2 above)
Comment (spiffcs) [reply cmd/cmd.go]: Updated to `anchoreLogger` since `log`, `logger`, and other names were taken

Comment (spiffcs) [reply cmd/cmd.go]: Updated to anchoreLogger since log, logger, and other names were taken

**Pull request merged**